pysatModels/models/sami2py_sami2.py

```diff
@@ -43,6 +44,9 @@
 # specify using xarray (not using pandas)
 pandas_format = False
 
+# Epoch origin for constructing time from sami files.
+_epoch_origin = dt.datetime(2019, 1, 1)
+
```

Comment:
This may be misleading, pinging @jklenzing for a sanity check.  

In normal SAMI runs, the epoch origin is actually the day of the model run.  Is that true in this case?  If so, I recommend removing this and using `_test_dates` instead.  If not, why was this date chosen instead of something more sensible, like the UNIX epoch?

Reply:
The date is a fake employed for pysat compatibility.  Current output of sami2py only includes UT, not date.  See https://github.com/sami2py/sami2py/issues/144

Reply:
I think sami2py is fine, @rstoneback just needs to make the appropriate changes here.